lualib/lua_magic/heuristics.lua

```diff
@@ -471,4 +471,20 @@ exports.pdf_format_heuristic = function(input, log_obj, pos, part)
   return 'pdf',weight
 end
 
+exports.pe_part_heuristic = function(input, log_obj, pos, part)
+  local content = part:get_raw_content()
```

Comment:
Why cannot we use `input` there?

Reply:
I failed to realise that `input` is what I wanted.